src/plugins/autologin/MQ2AutoLogin.cpp

```diff
@@ -617,6 +712,7 @@ PLUGIN_API void OnPulse()
 			{ "news",                 "NEWS_OKButton"}
 		};
 
+		memcpy_s(&pAutoLoginType->m_record, sizeof(std::optional<ProfileRecord>), (const void *)Login::getrecord(), sizeof(std::optional<ProfileRecord>));
```

Comment:
This isn't legal -- you can't memcpy a non-pod type. You should be able to just perform a straight assignment here.

Is this meant to capture the record that we're using to log in with? It might be better to assign this where we set the profile with SetLoginInformation

Reply:
Changed to assign directly via getter.